Library/ViewModels/OptimizelyFeatureFlagToolsViewModelTests.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+@testable import KsApi
+@testable import Library
+import Prelude
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import XCTest
+
+final class OptimizelyFlagToolsViewModelTests: TestCase {
+  private let vm: OptimizelyFeatureFlagToolsViewModelType = OptimizelyFeatureFlagToolsViewModel()
+
+  private let reloadWithData = TestObserver<OptimizelyFeatures, Never>()
+  private let updateUserDefaultsWithFeatures = TestObserver<OptimizelyFeatures, Never>()
+
+  override func setUp() {
+    super.setUp()
+
+    self.vm.outputs.reloadWithData.observe(self.reloadWithData.observer)
+    self.vm.outputs.updateUserDefaultsWithFeatures.observe(self.updateUserDefaultsWithFeatures.observer)
+  }
+
+  func testReloadWithData() {
```

Comment:
Maybe test a state where not all features are set to `true` in this test? Like (true, false, true).

Reply:
`testFeaturesEnabled_AfterReload_HasOptimizelyFeaturesEnabled`

Reply:
Going to use part of this rename but leave the prefix because I think we decided during the last native eng meeting that the tests names should be `test` + `output` + underscore delimited variables.

Reply:
Also added another test with a true, false, true mixture.